docs/library/overlay/toast.md

```diff
@@ -66,4 +66,21 @@ rx.button(
         style={"background-color": "green", "color": "white", "border": "1px solid green"}
     )
 )
-```
\ No newline at end of file
+```
+
+The following props are available for customization:
```

Comment:
Shouldn't these get auto-documented at the bottom? I think we can remove all this

Reply:
They're not component, so no.

We'd need to add additional logic to auto-document any class inheriting from `PropsBase`.